### PR TITLE
feat!: Update to Uno 5

### PR DESCRIPTION
**Pull request by juliecantin**
GitHub Issue: #

## Proposed Changes

Feature

## What is the current behavior?

Nventive.Persistence targets Uno 4 and .net 6, supports Xamarin.

## What is the new behavior?

Nventive.Persistence targets Uno 5 and .net 7, doesn't support Xamarin.

## Checklist

Please check if your PR fulfills the following requirements:

- [x] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ]  ~~Contains **NO** breaking changes~~
- [x] Updated the Changelog
- [ ] Associated with an issue

Apps will need to [migrate to Uno 5](https://platform.uno/docs/articles/migrating-to-uno-5.html).

### CODE REVIEW

**Review by jeanplevesque — APPROVED** [review]

File: src/Persistence.Uno.WinUI/Persistence.Uno.WinUI.csproj [hunk 1]
```diff
@@ -34,26 +34,22 @@
...
 	</ItemGroup>
 
-	<ItemGroup Condition="'$(TargetFramework)'=='net6.0-windows10.0.18362'">
+	<ItemGroup Condition="'$(TargetFramework)'=='net7.0-windows10.0.19041'">
 		<PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.3" />
 		<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.22000.24" />
 		<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="10.0.22000.24" />
 	</ItemGroup>
 
-	<PropertyGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
```
Comment (jeanplevesque) [inline src/Persistence.Uno.WinUI/Persistence.Uno.WinUI.csproj:43]: Keep WASM.

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

File: build/gitversion.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 assembly-versioning-scheme: MajorMinorPatch
 mode: ContinuousDeployment
-next-version: 0.4.0
+next-version: 1.0.0
```
Comment (jeanplevesque) [inline build/gitversion.yml]: This package is still in pre-release. The `next-version` should be 0.5.0

**Review by jeanplevesque — CHANGES_REQUESTED** [review]

**Review by CBrunet2212 — APPROVED** [review]

**Review by carlh98 — APPROVED** [review]

**Review by Soap-141 — APPROVED** [review]

**Comment by juliecantin** [comment]
@jeanplevesque We've implemented the changes you've requested, is the PR good to go?

**Comment by carlh98** [comment]
@juliecantin JP should be back tomorrow :)

**Review by jeanplevesque — APPROVED** [review]

**Pull request merged**